src/styles/custom.scss

```diff
@@ -153,15 +153,22 @@ iframe {
   box-shadow: 10px 10px 20px rgba(0, 0, 0, 0.1);
 }
 
-.git-link svg {
-  height: 1em;
-  width: 1em;
-  vertical-align: -0.1em;
-}
-.git-link path {
-  fill: currentColor;
+.git-link {
+  &::before {
+    content: '';
+    height: 1em;
+    width: 1em;
+    display: inline-block;
+    background-color: currentColor;
+    mask-repeat: no-repeat;
+    mask-position: center;
+    mask-size: 100% 100%;
+    mask-image: url(../../static/icons/code-branch.svg);
+    vertical-align: -0.1em;
+  }
```

Comment:
MDX breaks when there are new lines within tags.

Docusaurus [recommends](https://docusaurus.io/docs/next/markdown-features/react) to either use {/* prettier-ignore */} or add the files to .prettierignore until Prettier has support for MDX 3.

The previous codMDX breaks when there are new lines within tags.

Docusaurus [recommends](https://docusaurus.io/docs/next/markdown-features/react) to either use {/* prettier-ignore */} or add the files to .prettierignore until Prettier has support for MDX 3.

The [previous code](https://github.com/ionic-team/ionic-docs/pull/3322/files#diff-50a5b2c19690e36760631eb4db84135fc2b3e1e6de76ce00467a1f1f05ea9b87L18) would have the `<svg>` tag within the tag. The structure would break the site b/c of the new lines. Instead of going to every single file to fix it. I opted to moving the svg to the CSS.e would have the `<svg>` tag within the tag. The structure would break the site b/c of the new lines. Instead of going to every single file to fix it. I opted to moving the svg to the CSS.